### PR TITLE
Create patch_106_107_a.sql

### PR DESCRIPTION
**Pull request by jamie-m-a**
Updating schema to 107

### CODE REVIEW

**Comment by dglemos** [comment]
The schema version update is missing in sql/table.sql

**Comment by jamie-m-a** [comment]
Thanks - missed that! Added now

File: sql/patch_106_107_b.sql [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Change the transcript_variation & variation_feature tables to add new splice region consequences (splice_donor_5th_base_variant, splice_donor_region_variant and
```
Comment (dglemos) [inline sql/patch_106_107_b.sql:1]: This file is missing the header.

**Comment by dglemos** [comment]
PR looks good, the failing checks are not related to this PR.

**Review by dglemos — APPROVED** [review]

**Pull request merged**